btrack/__init__.py

```diff
@@ -1,3 +1,19 @@
-# __all__ = ['core','utils','constants','render']
```

Comment:
This looks like it's separate from this PR and might still be required?

Reply:
I think this is not required any more. It was commented out anyway, but... happy to take @dstansby advice on this!

Reply:
I'm 99% sure this isn't required - @paddyroddy why did you think it might be?

I'd suggest adding `__all__ = ['BayesianTracker'] though to prevent `logger` being documented in the API documentation.

Reply:
I have no idea, it just looked unrelated to the `logging` PR so was curious as to why it was being removed. That's all

Reply:
OK - I've changed this as suggested. Should be good now.